DDDEastAnglia.Tests/Sponsors/sponsor_list/Sponsors_who_havent_paid_yet.cs

```diff
@@ -0,0 +1,31 @@
+using System.Linq;
+using NUnit.Framework;
+
+namespace DDDEastAnglia.Tests.Sponsors.sponsor_list
+{
+    [TestFixture]
+    public sealed class Sponsors_who_havent_paid_yet : context
+    {
+        public Sponsors_who_havent_paid_yet()
```

Comment:
Question: should we make this a method and decorate it with `[TestFixtureSetUp]`?

The reason I ask is because I wasn't expecting to see this pattern. In a suite of Xunit-based tests, I think I'd have more quickly recognised this as being a constructor, but because it's an NUnit suite I was surprised not to see `[Test]` on this method. 

Not sure whether this is me being blind/stupid or a POLA thing.